lib/core/utils/preload.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Validated the preload object
+ * @param {Object | boolean} preload configuration object or boolean passed via the options parameter to axe.run
+ * @return {boolean}
+ * @private
+ */
+function isValidPreloadObject(preload) {
+	if (typeof preload !== 'object') {
+		return preload;
```

Comment:
preload=false shouldn't return false in `isValidPreloadObject`. This is misleading.